exercises/run-length-encoding/example.go

```diff
@@ -0,0 +1,55 @@
+package run_length_encoding
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func RunLengthEncode(s string) string {
+	count := 1
+	output := ""
```

Comment:
It's more idiomatic to write `var output string` if you're going to initialise a variable to it's zero value